js/apps/system/_admin/aardvark/APP/react/src/utils/constants.ts

```diff
@@ -23,8 +23,13 @@ export type DispatchArgs<FormState> = {
   formState?: FormState;
 };
 
+export type RenderView = "LinkList" | "AddNew" | "ViewParent" | "ViewChild" | "ViewField";
```

Comment:
This is views-specific, and should be part of `views/constants.ts`.

Reply:
Done.